std/assembly/dataview.ts

```diff
@@ -39,7 +42,7 @@ export class DataView {
     if (
       i32(byteOffset < 0) |
       i32(byteOffset + 4 > this.dataLength)
-    ) throw new Error("Offset out of bounds");
+    ) throw new RangeError(E_INDEXOUTOFRANGE);
```

Comment:
how about make special instance? and use it as `throw new IOOFError()`?

Reply:
Not sure about the benefits, as checking for `RangeError` usually suffices and is in line with the spec. Suggesting to think about this again once exceptions are actually supported :)